package.json

```diff
@@ -7,10 +7,9 @@
   "scripts": {
     "build": "tsc-esm && tsc -p tsconfig-cjs.json && find dist/cjs -type d -execdir sh -c 'cd {}; mmv -rd \"*.js*\" \"#1#2\"' \\;",
     "test": "jest",
-    "prepublish": "rm -rf dist && yarn format && yarn lint && yarn build && yarn test",
+    "prepublish": "rm -rf dist && yarn lint && yarn build && yarn test",
     "lint": "eslint src --ext .ts",
-    "format": "prettier -c \"src/**/*.{ts,tsx,js}\"",
-    "format:fix": "prettier -w \"src/**/*.{ts,tsx,js}\""
+    "lint:fix": "eslint --fix --ext .ts src"
```

Comment:
`format`?

Reply:
I added the eslint plugin for prettier. Therefore lint checks the code format using prettier and lint:fix reformats the code.